sources/server/Server.cpp

```diff
@@ -6,7 +6,7 @@
 /*   By: mvieira- <mvieira-@student.42sp.org.br>    +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/01/03 10:16:53 by mvieira-          #+#    #+#             */
```

Comment:
Delete 42 headers, as they clutter the git diffs with updates

Reply:
Removed all headers